packages/sdk-metrics/src/Meter.ts

```diff
@@ -51,7 +51,6 @@ export class Meter implements IMeter {
 
   /**
    * Create a {@link Gauge} instrument.
-   * @experimental
    */
   createGauge<AttributesTypes extends Attributes = Attributes>(
```

Comment:
This is implicitly true via `IMeter`
```suggestion
  createGauge(
```